### ISSUE TITLE
Use nix to build executable

### ISSUE THREAD

**Issue by turion** [MaxOw/RuneSlayer#1]
It would be cool if the last step of the build (`cabal build`) is already done, so you can just do `nix-build` and it will build everything, including the executable.

**Comment by MaxOw** [comment]
Actually you can already do that. I should probably write it explicitly in README.

**Comment by MaxOw** [comment]
Fixed in f579f46. Thanks!

**Comment by turion** [comment]
Unfortunately I can't get this to work. At the very end it says:

```
src/Version.hs:31:23: error:
    • Exception when trying to run compile-time code:
        git: readCreateProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
      Code: Git.embedGitRevision
    • In the untyped splice: $(Git.embedGitRevision)
   |
31 | revision = fromString $(Git.embedGitRevision)
   |                       ^^^^^^^^^^^^^^^^^^^^^^^
```

Building with `nix-shell` and `cabal` works.

**Comment by MaxOw** [comment]
Huh? Funny thing is that before answering you here I did test that it worked. But now I did a git pull on that machine and it stopped? Anyways the problem is that the git is missing from environment so I'll fix that in a sec. I'm just surprised that it somehow worked before.

**Comment by MaxOw** [comment]
Fixed in 56f7647. I hope.
